app/routes/cdo/edit/change-status.js

```diff
@@ -54,11 +72,16 @@ module.exports = [
     options: {
       auth: { scope: anyLoggedInUser },
       validate: {
-        payload: validatePayload,
+        payload: validateChangeStatusPayload,
         failAction: changeStatusPostFailAction
       },
       handler: async (request, h) => {
         const payload = request.payload
+        const backNav = addBackNavigation(request, false, true)
+
+        if (payload.newStatus === 'In breach') {
+          return h.redirect(`${routes.inBreach.get}/${payload.indexNumber}?src=${backNav?.srcHashValue}`)
```

Comment:
I think this redirect may lose the back nav (or at least make it work wrongly) as you're creating a new srcHashParam. Worth testing this specific back nav case

Reply:
yeah, we tested this one on the screenshare.